.nvmrc

```diff
@@ -1 +1 @@
-10
+12
```

Comment:
Gatsby v3 requires a minimum Node.js version of 12.13.0: https://www.gatsbyjs.com/docs/reference/release-notes/migrating-from-v2-to-v3/#minimal-nodejs-version-12130